File: laprak6/PRAK505-2210817210032-fransiskusassisiindra.py
def biodata(tahunlahir,A,B):
    tahun_sekarang = 2022
    umur = tahun_sekarang - tahunlahir
    print('\nPerkenalkan Nama Saya : ' + A)
    print("Umur Saya : %d" %(umur))
    print('Saya Adalah Angkatan :', tahun_sekarang)
    print('Asal Saya Dari : '+ B )
tahunlahir = int(input())
A = str(input())
B = str(input())
biodata(tahunlahir, A, B)

File: laprak6/PRAK505-2210817210032-fransiskusassisiindra.c
#include <stdio.h>
void biodata(int tahunlahir, char *Namaku, char *Asal){
int tahun_sekarang = 2020;
printf("\nPrerkenalkan Nama Saya : %s\n", Namaku);
printf("Umur Saya : %d\n",tahun_sekarang - tahunlahir);
printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
printf("Asal saya dari : %s", Asal);}
int main() {
int tahunLahir;
char Namaku[20], Asal[15];
scanf(" %d",&tahunLahir);
scanf(" %s[^\n]%*c",&Namaku);
scanf(" %s[^\n]%*c",&Asal);
biodata(tahunLahir, Namaku, Asal);
return 0;}
